app/views/application_mailer/office_application_notification.html.erb

```diff
@@ -0,0 +1,3 @@
+Hello!
+
+A new application has been submitted and is attached here.
```

Comment:
I know this is just placeholder text, but perhaps we should indicate that it's a SNAP application?